includes/class-stats.php

```diff
@@ -185,4 +187,126 @@ public function maybe_log_listing_view() {
 
 		$this->log_stat( 'job_listing_view', [ 'post_id' => $post_id ] );
 	}
+
+	/**
+	 * Log multiple stats in one go. Triggered in an ajax call.
+	 *
+	 * @return void
+	 */
+	public function maybe_log_stat_ajax() {
+		if ( ! ( defined( 'DOING_AJAX' ) || ! DOING_AJAX ) ) {
```

Comment:
WDYT about using the proper function for this?

```suggestion
		if ( ! wp_doing_ajax() ) {
```